signalfx/resource_signalfx_alert_muting_rule.go

```diff
@@ -0,0 +1,256 @@
+package signalfx
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	alertmuting "github.com/signalfx/signalfx-go/alertmuting"
+)
+
+func alertMutingRuleResource() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"description": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Description of the rule.",
```

Comment:
This is somewhat pedantic but none of these descriptions are sentences -- so the period really should just be omitted.

Reply:
Fixed!